### PR TITLE
[doc] Add table docs with hide, archive and delete functionality

### PR DESCRIPTION
**Pull request by VkoHov**

### CODE REVIEW

File: docs/source/ui/pages/table.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+The table helps to explore and compare training runs results, logged hyper-parameters, system info, etc. As well as to apply quick actions (hide, delete, archive, etc.) over a batch of selected items within a few clicks.
+
+### Delete a batch of items
+
+To delete a batch of runs select rows and click on the `Delete` button, which appears at the top of the table.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Delete a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items.png">
+
+.. note::
+In-progress runs can't be deleted. Wait until the process is finished or terminate the progress to be able to delete them.
```
Comment (gorarakelyan) [inline docs/source/ui/pages/table.md:14]: terminate the process I guess

File: docs/source/ui/pages/table.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+After clicking on the `Delete` button a popup appears where `finished` and `in-progress` runs are divided into a separate tables.
```
Comment (gorarakelyan) [inline docs/source/ui/pages/table.md:16]: divided into separate tables

File: docs/source/ui/pages/table.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup.png">
+
+To undo a selected run from the selected list click on the `X` icon which appears when a row is hovered over.
```
Comment (gorarakelyan) [inline docs/source/ui/pages/table.md:20]: To undo a selection, click on the `X` icon which appears when a row is hovered over.

File: docs/source/ui/pages/table.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup remove run" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup_remove_run.png">
+
+### Archive/Unarchive a batch of items
+
+To archive/unarchive a batch of runs select rows and click on the `Archive/Unarchive` button, which appears at the top of the table.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Archive a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/archive_a_batch_of_items.png">
+
+After clicking on the `Archive/Unarchive` button a popup appears, where `unarchived` and `archived` runs are divided into a separate tables.
```
Comment (gorarakelyan) [inline docs/source/ui/pages/table.md:30]: where `archived` and `unarchived` runs? you always mentioned archived runs first, would be better to stick to the same order

File: docs/source/ui/pages/table.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+## Table
+
+### Overview
+
+The table helps to explore and compare training runs results, logged hyper-parameters, system info, etc. As well as to apply quick actions (hide, delete, archive, etc.) over a batch of selected items within a few clicks.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:5]: The Aim Table helps to explore and compare training runs - their results, logged hyper-parameters, system info, etc. You can also use the table to hide, delete, archive and compare selected training runs in a few clicks.

File: docs/source/ui/pages/table.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+### Delete a batch of items
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:7]: Deleting selected runs

File: docs/source/ui/pages/table.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+To delete a batch of runs select rows and click on the `Delete` button, which appears at the top of the table.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:9]: ... which is located at the top of the table.

File: docs/source/ui/pages/table.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Delete a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items.png">
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:11]: alt="Delete a set of selected runs"

File: docs/source/ui/pages/table.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+.. note::
+In-progress runs can't be deleted. Wait until the process is finished or terminate the process to be able to delete them.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:14]: ... can't be deleted. Please wait until the run is finished or terminate it to be able to delete the run.

File: docs/source/ui/pages/table.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+After clicking on the `Delete` button a popup appears where `finished` and `in-progress` runs are divided into separate tables.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup.png">
+
+To undo a selection, click on the X icon which appears when a row is hovered over.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:20]: To undo the selection, hover over the row. Then the X button will appear, press the X button.

File: docs/source/ui/pages/table.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup remove run" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup_remove_run.png">
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:22]: alt="The popup for deleting the selected runs"

File: docs/source/ui/pages/table.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+### Archive/Unarchive a batch of items
+
+To archive/unarchive a batch of runs select rows and click on the `Archive/Unarchive` button, which appears at the top of the table.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:26]: ...archive/unarchive a set of runs ... which is located at the top of the table...

File: docs/source/ui/pages/table.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+After clicking on the `Delete` button a popup appears where `finished` and `in-progress` runs are divided into separate tables.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup.png">
+
+To undo a selection, click on the X icon which appears when a row is hovered over.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup remove run" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup_remove_run.png">
+
+### Archive/Unarchive a batch of items
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:24]: Archive / Unarchive runs

File: docs/source/ui/pages/table.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+To archive/unarchive a batch of runs select rows and click on the `Archive/Unarchive` button, which appears at the top of the table.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Archive a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/archive_a_batch_of_items.png">
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:28]: alt="Archive the selected runs"

File: docs/source/ui/pages/table.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+
+After clicking on the `Archive/Unarchive` button a popup appears, where `archived` and `unarchived` runs are divided into separate tables.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Archive a batch of items popup" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/archive_a_batch_of_items_popup.png">
+
+To undo a selection, click on the X icon which appears when a row is hovered over.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:34]: To undo the selection, hover over the selected row and click on the X button (appears on hover).

File: docs/source/ui/pages/table.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Archive a batch of items popup remove run" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/archive_a_batch_of_items_popup_remove_run.png">
+
+### Hide and show a batch of items
+
+To hide/show a batch of items select rows and click on the `Hide/Show Lines` button, which appear at the top of the table.
```
Comment (SGevorg) [inline docs/source/ui/pages/table.md:40]: ... located at the top of the table.

**Review by SGevorg — CHANGES_REQUESTED** [review]

**Review by SGevorg — COMMENTED** [review]
lgtm

**Review by gorarakelyan — APPROVED** [review]

**Review by SGevorg — APPROVED** [review]
lgtm

**Pull request merged**